content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: blog
+title: "Moving cgroup v1 Support into Maintenance Mode in Kubernetes 1.31"
+date: 2024-08-15
+slug: moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31
+author: Harshal Patil
+---
+
+As Kubernetes continues to evolve and adapt to the changing landscape of
```

Comment:
To be honest, we are transition to cgroup v2 due to the linux community evolving/adapting. 

Maybe we can say "As container orchestration evolves and adapts, the Kubernetes community has decided to move cgroup v1 support into maintance mode".